javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,47 @@
+//student manager
+const class07Students = [];
+function addStudentToClass(studentName) { 
+    if(studentName==='Queen') {
+        class07Students.push(studentName);
+    }
+  else if(studentName===''){
+      console.log('Please enter a valid name');
+  }
+  
+  else if(class07Students.length>=6){
+      console.log('Cannot add more students to class 07');
+  }
+  else if(toCheckStudentExist(studentName)){
+      console.log('Student '+ studentName+' is already in the class');
+  }  
+  else{
+      class07Students.push(studentName)
+      return class07Students;
```

Comment:
you already have declared your array global...therefore you don't need to return.
```suggestion
```

Reply:
I just removed the return statement.